### PR TITLE
Add order_id based scope resolve for admin sales module

### PR DESCRIPTION
**Pull request by silverman63**
This resolves a bug where emails sent via the admin sales module weren't properly retrieving the correctly scoped module settings. Previously, the scope was resolved by `store` or `website` request params or through `StoreManager`. We've now added a check for the `order_id ` request param which is exposed in the admin sales module and can look up store ID through that request.

### CODE REVIEW

File: Helper/AbstractHelper.php [hunk 1]
```diff
@@ -46,24 +47,34 @@ public function __construct(
...
+        if ($websiteCode = $this->_request->getParam('website')) {
+            return $this->scopeConfig->getValue($val, ScopeInterface::SCOPE_WEBSITE, $websiteCode);
+        }
+
+        return $this->scopeConfig->getValue($val, ScopeInterface::SCOPE_STORE, $this->storeManager->getStore()->getCode());
+    }
+
+    private function getStoreCodeFromOrderId($orderId) {
+        /** @var CollectionFactoryInterface $orderFactory */
+        $orderFactory = $this->objectManager->create('\Magento\Sales\Model\ResourceModel\Order\CollectionFactoryInterface');
```
Comment (sshymko) [inline Helper/AbstractHelper.php]: Order repository injected as a dependency on the constructor should be used.

Comment (silverman63) [reply Helper/AbstractHelper.php]: Are you saying 1) use order repo instead of collection, and 2) use DI?

Comment (sshymko) [reply Helper/AbstractHelper.php]: Yes, both 1) and 2), for instance:
```php
public function __construct(
    OrderRepositoryInterface $orderRepository
) {
    $this->orderRepository = $orderRepository;
}

// ...

$order = $this->orderRepository->get($orderId);
```

**Comment by silverman63** [comment]
Closing, addressed more comprehensively by #41

**Pull request closed without merging**